.github/workflows/ci_frontend.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           # process.env.BRANCH = 'pull_27';
           echo "BRANCH=$(echo ${BRANCH_NAME})" >> $GITHUB_ENV
 
-      - if: steps.changes.outputs.src == 'true' && github.event_name != 'pull_request'
```

Comment:
This condition was added from your previous PR to resolve the old issue. Wouldn't remove it now bring back that error?

Reply:
As it turned out, this step is not what causes the failure, only the PR comment step below on line 86 is the one that causes it